docs/developers.md

```diff
@@ -38,6 +38,21 @@ How to import a Gradle project:
 * [How to import Gradle project into IntelliJ IDEA](https://www.jetbrains.com/help/idea/gradle.html) (section "Importing a project
   from a Gradle model").
 
+**Note for Eclipse Users:**
+
+Eclipse needs to have Gradle build the generated source files before it can build the project.
```

Comment:
What about replacing

> Eclipse needs to have Gradle build the generated source files before it can build the project.

with something like

> To succesfuly import the project in Eclipse you need complete project structure, incl. files built usually at compile time, therefore you need to either build the whole project or at least run `genBuildInfo` Gradle task first.

Reply:
Worded differently.